commands/me.js

```diff
@@ -25,8 +25,15 @@ module.exports = {
 		if (((profileData.xpTimeoutUntil - message.createdTimestamp > 0) && (!configData.xp.xpTimeoutHidden)) || (override)) {
 			fields.push({ name: "XP Timeout", value: functions.msToString(profileData.xpTimeoutUntil - message.createdTimestamp), inline: true });
 		}
-		let xpPercentage = Math.round(profileData.xp / Math.pow(profileData.level + configData.xp.levelBaseOffset, configData.xp.levelExponent) * 100);
+
+		let xpPercentage;
+		if (profileData.xp < 0) {
+			xpPercentage = 0;
+		} else {
+			xpPercentage = Math.round(profileData.xp / Math.pow(profileData.level + configData.xp.levelBaseOffset, configData.xp.levelExponent) * 100);
```

Comment:
Shouldn't the XP be set to 0 as well? Since negative XP values don't make any real sense??